gen/gen.go

```diff
@@ -60,7 +60,7 @@ func init() {
 		dirExercise = filepath.Join(path, "..", "..")
 	}
 	if dirExercise == "" {
-		dirExercise = ".."
+		dirExercise = "."
```

Comment:
you put this change into the README commit, though it should probably go in the "fix path" commit?